packages/ui-link/src/Link/styles.ts

```diff
@@ -50,6 +50,7 @@ const generateStyle = (
     fontWeight: componentTheme.fontWeight,
     transition: 'outline-color 0.2s',
     verticalAlign: 'baseline',
+    borderRadius: '0.25rem',
```

Comment:
I would say that it looks better with 0.125rem radius, it is more consistent with the CondensedButton too. Also, don't hardcode it, make it come from the theme's small border radius variable.
<img width="655" alt="image" src="https://user-images.githubusercontent.com/11446033/224314960-266d9d6c-4693-41d3-9847-44bad30ca9f4.png">